app/assets/stylesheets/stream_element.scss

```diff
@@ -38,12 +38,20 @@
 }
 
 #main_stream .stream_element {
-  padding: 10px;
   margin-bottom: 20px;
-  background-color: $white;
   border: 1px solid $light-grey;
   box-shadow: $card-shadow;
 
+  &.highlighted {
+    border-left: 3px solid $brand-primary;
+    padding-left: 8px;
+  }
+}
+
+.stream_element {
```

Comment:
There is another `.stream_element` rule directly below this one so you can merge both.